client/starwhale/core/instance/cli.py

```diff
@@ -28,15 +29,25 @@ def _select(instance: str) -> None:
 
 @instance_cmd.command("login")
 @click.argument("instance", default="")
-@click.option("--username", prompt="username", required=True)
-@click.password_option(confirmation_prompt=False)
+@click.option("--username")
+@click.option("--password")
+@click.option("--token", help="Get token from Starwhale Cloud page")
```

Comment:
Would it have been better if the word `Get` was removed?

Reply:
ok